### ISSUE TITLE
Claims are not showed in applications claim button pop-up

### ISSUE THREAD

**Issue by aureamunoz** [halkyonio/primaza-poc#390]
Giving a cluster registered in Primaza and an atomic-fruits application deployed in that cluster (It is listed in Applications page), we create via `curl` some elements such as db service with its corresponding credential. Then, we create also a Claim for this service and application via `curl`. From the root of the project run the following commands:
```
# Register this service as installable to let crossplane to install it
./scripts/data/services.sh url=localhost:8080 service_name=postgresql version=14.5 installable=on type=postgresql endpoint=tcp:5432 helm_repo="https://charts.bitnami.com/bitnami&helmChart=postgresql&helmChartVersion=11.9.13"

./scripts/data/credentials.sh url=localhost:8080 credential_name=fruits_database-vault-creds service_name=postgresql vault_kv=primaza/fruits

./scripts/data/claims.sh url=localhost:8080 claim_name=fruits-claim description=postgresql-fruits-db requested_service=postgresql-14.5

```
Then, go to primaza in a browser and click on the Àpplications button on the home page. The atomic-fruits is present: 
<img width="1074" alt="Captura de pantalla 2023-08-29 a las 15 03 32" src="https://github.com/halkyonio/primaza-poc/assets/6627952/8a02bdf2-3243-48d6-9f9f-75d0dd890faf">

Click on the `Claim` button. The pop-up is open with no claims on the selection list.

**Comment by cmoulliard** [comment]
Is there an error reported by the quarkus application during the execution of the bash scripts + curl and fianlly when you open the application screen ?
Is this problem similar to what we investigated and reason why the job failed as applicationId was null ?

@aureamunoz

**Comment by aureamunoz** [comment]
I think the problem is that the application id is not passed when creating the claim via ./scripts/data/claims.sh. When created in the UI, it is passed by default in the pop up opened from the application page.

**Comment by cmoulliard** [comment]
> I think the problem is that the application id is not passed when creating the claim via ./scripts/data/claims.sh.

Correct. So do we have to update the logic of the curl request within the script to be aligned with the java code ?
```
    @GET
    @Path("/new")
    @Produces(MediaType.TEXT_HTML)
    public TemplateInstance claim(@QueryParam("applicationId") Long applicationId) {
        Map<String, Object> optional = new HashMap<>();
        if (applicationId != null) {
            optional.put("applicationId", applicationId);
        }

        return Templates.Claims.form("Claim form", new Claim(), Service.listAll(), optional);
    }
```

@aureamunoz

**Comment by aureamunoz** [comment]
No, I don't think so, we need to pass the applicationId in the body when creating a claim via curl.

**Comment by cmoulliard** [comment]
> No, I don't think so, we need to pass the applicationId in the body when creating a claim via curl.

So this is `yes` as we must change the curl logic ;-) as today we don't pass the applicationId: See:  https://github.com/halkyonio/primaza-poc/blob/83765c0033e62734ae76c4ae123b271e6da1e52b/scripts/data/claims.sh#L58

@aureamunoz

**Comment by aureamunoz** [comment]
ah, ok, I hadn't read correctly your comment. What I meant is that there is not an error actually, when the scenario runs in the CI everything is fine. It's a bit confusing just when we use the commands to perform tests manually. We can update the script to be aligned with the workflow that binds from the applications page.

**Comment by cmoulliard** [comment]
I think that we should do that to be sure that CI or demo or Java Test cases are aligned together ;-)

**Comment by aureamunoz** [comment]
Fixed by #394
